apps/ts-minbar-test-react-components/assets/tsconfig.json

```diff
@@ -0,0 +1,14 @@
+{
+  "compilerOptions": {
+    "jsx": "react",
+    "lib": ["es2015", "dom"],
+    "target": "es5",
+    "strict": true,
+    "declaration": true,
+    "experimentalDecorators": true,
+    "isolatedModules": true,
+    "typeRoots": ["./typings", "./node_modules/@types"],
+    "types": ["custom-global", "inline-style-expand-shorthand"]
```

Comment:
Remove these. You shouldn't override typeRoots in the test, and types should just be `[]` (if `types` is totally unspecified it will automatically include everything under `node_modules/@types` in the compilation, which can slow things down a lot)
```suggestion
    "types": []
```

Reply:
I actually added this to fix this error, although i dont really need "inline-style-expand-shorthand" :
![image](https://user-images.githubusercontent.com/8649804/139507920-37984392-45e3-4db4-b886-6c7d1eaaaf89.png)

Reply:
Hmm I think that's actually a bug...we definitely need to avoid letting `custom-global` leak into our public types

Reply:
The custom-global leak is because of the override definitions for Map, which are getting pulled into that file because it references Map. I confirmed locally that this will be fixed after #20405 which adds a reference to `lib: ES2019` which includes the real Map.